### ISSUE TITLE
output from backends() is not correct. 

### ISSUE THREAD

**Issue by jaygambetta** [Qiskit/qiskit-jku-provider#21]
This is a follow on from #19. 

<img width="662" alt="image" src="https://user-images.githubusercontent.com/15731369/52981465-f720ee00-33ad-11e9-8157-017df184326c.png">

and it should be 

[<QasmSimulator('qasm_simulator') from JKUProvider()>]

**Comment by jaygambetta** [comment]
Actually, it goes to a few places. 

```
jku_backend = JKUProvider().get_backend('local_statevector_simulator_jku')
```

<img width="1007" alt="image" src="https://user-images.githubusercontent.com/15731369/52981965-24bb6680-33b1-11e9-80b9-3afc4d453ff4.png">

name should be qasm_simulator
and Provider is set to None

**Comment by gadial** [comment]
The `None` provider error is fixed. I'm not sure why `jku_backend.name()` should be **qasm_simulator**; I thought the naming convention implies that in this case we should specify both that this is a statevector simulator and that it is based on the JKU code.

**Comment by gadial** [comment]
Ok, reading #20 I understand your line of thought. Will fix.

**Comment by gadial** [comment]
Closing as this is addressed in #37
